src/client/app/pages/DetailsPage/data/DetailsPageActions.js

```diff
@@ -0,0 +1,31 @@
+import {fetchMovie} from '../../../api/api';
+import {DETAILS_ACTIONS} from './consts';
+
+export function getMovie (id) {
+  if (id) {
```

Comment:
It's bad style to mix different type of actions in one action creator. Why does this check exist here?